PC/VS9.0/readme.txt

```diff
@@ -132,7 +132,7 @@ _ssl
 
     Get the source code through
 
-    svn export http://svn.python.org/projects/external/openssl-1.0.2k
+    svn export http://svn.python.org/projects/external/openssl-1.0.2o

```

Comment:
It's not worth changing in this PR but we no longer pull externals from svn.python.org, which is soon to be no more.  I would be happy to accept another PR that fixes the readme (and also make sure PCbuild\readme.txt is correct in all branches), though.